src/commands/init.js

```diff
@@ -7,6 +7,64 @@ const fs = require('fs-extra')
 const { installDeps } = require('../util')
 const defaultAPMName = require('../helpers/default-apm')
 
+
+const handlerTasks = {
+  checkProjectExistence: (options) => {
```

Comment:
`handlerTasks` is exported below so tasks can be imported individually when testing.

Also, each handler task receives options. That's because a command's `handler` might introduce some additional logic and properties that may be needed within handler tasks.

Reply:
I think in general `options` should be replaced by the actual option values that this particular task needs, in this case: `options` -> `{ basename }`.

Reply:
Yes makes sense. It probably makes sense to find a general API that covers all possible cases, so that, whenever a new CLI functions is introduced, they expect the same parameters etc. Will have to look at other tasks to see what exactly would work. But we can definitely just go with destruction patterns for now.